repository/t.json

```
@@ -2653,6 +2653,17 @@
 				}
 			]
 		},
+		{
+			"name": "Tmux Syntax",
```

```suggestion
			"name": "Tmux Syntax Highlight",
```

Now that the 2 weeks are over and we've heard nothing, let's go ahead with the replacement. You can rename your package and then remove the existing entry for "Tmux Syntax Highlight".